tools/Benchmarks/Benchmarks.cs

```diff
@@ -105,7 +105,7 @@ private static (int Min, int Max) GetMinMax(ResultType resultType)
                 break;
             case ResultType.Mixed:
                 min = 0;
-                max = 1;
+                max = 2;
```

Comment:
`random.Next()`'s max parameter is exclusive, so I upped this to 2 - your existing benchmarks for mixed results would've been the same as those for All OK